src/main/kotlin/com/sourcegraph/cody/config/actions/OpenCodySettingsEditorAction.kt

```diff
@@ -0,0 +1,72 @@
+package com.sourcegraph.cody.config.actions
+
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.diagnostic.Logger
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.vfs.LocalFileSystem
+import com.intellij.util.io.write
+import com.jetbrains.jsonSchema.JsonSchemaMappingsProjectConfiguration
+import com.jetbrains.jsonSchema.UserDefinedJsonSchemaConfiguration
+import com.jetbrains.jsonSchema.ide.JsonSchemaService
+import com.jetbrains.jsonSchema.impl.JsonSchemaVersion
+import com.sourcegraph.cody.agent.CodyAgentService
+import com.sourcegraph.common.ui.DumbAwareEDTAction
+import com.sourcegraph.config.ConfigUtil
+import com.sourcegraph.utils.CodyEditorUtil
+import kotlin.io.path.name
+
+class OpenCodySettingsEditorAction : DumbAwareEDTAction("Open Cody Settings Editor") {
+  override fun actionPerformed(e: AnActionEvent) {
+    val project =
+        e.project
+            ?: run {
+              logger.warn("Project cannot be null")
+              return
+            }
```

Comment:
we can log it but it looks like a common practice is to simply `?: return` (see our other actions and actions in IntelliJ)